app/controllers/events/email.py

```diff
@@ -0,0 +1,36 @@
+from flask import Flask, redirect, flash, url_for, request, g
+from flask_mail import Mail, Message
+from app.models.interest import Interest
+from app.models.user import User
+from app.models.eventParticipant import EventParticipant
+from app.logic.emailHandler import getInterestedEmails, getParticipantEmails, emailHandler
+from app.controllers.events import events_bp
+from app import app
+
+@events_bp.route('/email', methods=['POST'])
+def emailVolunteers():
+    """ Uses emailHandler to send an email with the form in event_list. """
+    emailInfo = request.form
+
+    if emailInfo['emailRecipients'] == "interested":
+        emails = getInterestedEmails(emailInfo['programID'])
+    elif emailInfo['emailRecipients'] == "eventParticipant":
+        emails = getParticipantEmails(emailInfo['eventID'])
+    else:
+        flash("Unable to determine email recipients", "danger")
+
+    if emails == None:
+        flash("Error getting email recipients", "danger")

```

Comment:
You're going to double flash here if there are no emails. I'd remove the `else` condition above, since you are handling the error case here.